BGL/doc/BGL/CGAL/boost/graph/Named_function_parameters.h

```diff
@@ -0,0 +1,11 @@
+namespace CGAL{
```

Comment:
@CGAL/geometryfactory here are the new function and type that we propose to add (to replace `all_default()`). These will then be used in the function added in the test below.

Reply:
The goal to do this change is readability only or there is smth else?

Reply:
readibility + doc of the default type so that the code can be simplified and avoid overload calling the function with `all_default()` as c++11 allows default value for template parameters of functions